providers/attio/types.go

```diff
@@ -0,0 +1,11 @@
+// nolint
+package attio
+
+import "errors"
+
+var ErrEmptyResultResponse = errors.New("writing reponded with an empty result")
+
+type writeResponse struct {
```

Comment:
I'd move all these to `write.go` for the same rationale as my other comment, especially since they also are all about writing